client/apps/edit/components/content/sections/text/index.coffee

```diff
@@ -7,7 +7,20 @@ Config = require './draft_config.js'
 Nav = React.createFactory require '../../../../../../components/rich_text/components/nav.coffee'
 InputUrl = React.createFactory require '../../../../../../components/rich_text/components/input_url.coffee'
 Text = React.createFactory Text
-Utils = require '../../../../../../components/rich_text/utils/index.coffee'
+{
+  setSelectionToStart,
+  stickyControlsBox,
+  getSelectedLinkData,
+  getSelectionDetails
+} = require '../../../../../../components/rich_text/utils/text_selection.js'
+{ setContentEnd } = require '../../../../../../components/rich_text/utils/decorators.js'
+{
+  standardizeSpacing,
+  stripCharacterStyles,
+  stripGoogleStyles
+} = require '../../../../../../components/rich_text/utils/text_stripping.js'
+{ convertFromRichHtml, convertToRichHtml } = require '../../../../../../components/rich_text/utils/convert_html.js'
+{ keyBindingFnFull } = require '../../../../../../components/rich_text/utils/keybindings.js'
```

Comment:
These imports are long, but will be fixed up in a coming PR refactoring this component.